cmake/googletest_download.txt.in

```diff
@@ -14,14 +14,6 @@ set(CMAKE_CXX_STANDARD 11)
 set(CMAKE_CXX_EXTENSIONS OFF)
 set(CMAKE_CXX_STANDARD_REQUIRED ON)
 
-set(FORCE_SHARED_CRT "")
-set(FORCE_OLD_ABI "")
-if(MSVC)
-  set(FORCE_SHARED_CRT "-DFORCE_SHARED_CRT=OFF")
```

Comment:
I get the change for Linux (CentOS 8), but why this change to Windows?

Reply:
It is the same issue; this line was never taken into account so I decided to remove it.